TumbleBitSetup.Tests/ProtocolTests.cs

```diff
@@ -380,8 +379,8 @@ public bool _DifferentETest(int keySize)
             // The different public key to be used in verifying.
             var secPubKey = new RsaPubKey(diffKey);
 
-            byte[][] signature = PermutationTest.Proving(privKey.P, privKey.Q, privKey.PublicExponent, alpha, ps);
-            return PermutationTest.Verifying(secPubKey, signature, alpha, keySize, ps);
+            var signature = keyPair.CreatePermutationTestProof(setup);
+            return secPubKey.VerifyPermutationTestProof(signature, setup);
```

Comment:
Maybe we should be consistent with using "diffKey" like in "_DifferentNTest" instead of "secPubKey"